polyfills/Element/prototype/classList/config.json

```diff
@@ -4,7 +4,7 @@
 		"modernizr:classlist"
 	],
 	"browsers": {
-		"ie": "6 - 9",
+		"ie": "8 - 9",
```

Comment:
I think that this on its own would count as a breaking change. Should we put out an RFC to see if this is something people are okay with? (I have no idea how Polyfill-Service orchestrates dropping support)